docs/specviz/displaying.rst

```diff
@@ -51,6 +51,15 @@ on the top right of the viewer:
 
 .. _image-pan-zoom:
```

Comment:
This ref link needs to be moved to "2D Bidirectional Pan/Zoom". It now links to the wrong section.

Reply:
Ok, I moved it back.  I originally was thinking that the link should go to the top of that entire section, but I agree it makes more sense to go directly to pan/zoom.